src/main/java/com/minecolonies/coremod/commands/BackupCommand.java

```diff
@@ -0,0 +1,74 @@
+package com.minecolonies.coremod.commands;
+
+import com.minecolonies.coremod.colony.ColonyManager;
+import net.minecraft.client.Minecraft;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.TextComponentString;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by asie on 2/16/17.
+ */
+public class BackupCommand extends AbstractSingleCommand
+{
+    public static final String DESC = "backup";
+    public static final String NO_PERMISSION_MESSAGE = "You do not have permission to backup colony data!";
+    public static final String BACKUP_SUCCESS_MESSAGE = "Successfully backed up colony data!";
+    public static final String BACKUP_FAILURE_MESSAGE = "Failed to back up colony data!";
+
+    /**
+     * Initialize this SubCommand with it's parents.
+     *
+     * @param parents an array of all the parents.
+     */
+    public BackupCommand(@NotNull final String... parents)
+    {
+        super(parents);
+    }
+
+    @Override
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
+    {
+        if (!sender.canCommandSenderUseCommand(4, "minecolonies backup"))
```

Comment:
@asiekierka We need this to be 3 it seems. There should be a constant somewhere in the commands hirarchy for that. I guess even a config option.

mc is weird xD